gossip/src/crds.rs

```diff
@@ -56,6 +57,26 @@ use {
 const CRDS_SHARDS_BITS: u32 = 12;
 // Number of vote slots to track in an lru-cache for metrics.
 const VOTE_SLOTS_METRICS_CAP: usize = 100;
+// Number of ending zero bits for crds signature to get reported to influx
+// mean new push messages received per minute per node
+//      testnet: ~500k,
+//      mainnet: ~280k
+// target: 1-2 signatures reported per minute
+// log2(250k) = ~17.9.
+const SIGNATURE_SAMPLE_TRAILING_ZEROS: u8 = 18;
```

Comment:
The comment says "ending zero bits" and the constant is named `_TRAILING_`.
But seems like the implementation is looking at leading zeros.

Reply:
ok so i agree but this: https://docs.rs/ed25519/1.0.2/src/ed25519/lib.rs.html#272 says "at least the three **highest** bits of the scalar will be unset" and then it checks 
```
if bytes[SIGNATURE_LENGTH - 1] & 0b1110_0000 != 0
```
it's looking at the end of the bytes array for the MSB. so this would imply that bytes[0] is the LSB? or am I misunderstanding?

But i guess this doesn't matter if we use what you suggested below

Reply:
oh wait i see what you are saying, you're right. I am comparing against the wrong end of the signature. ignore above. but the conclusion is same. we don't need tis if we're using what you suggest below